STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.filter.boundary.FilterContainerTest.testIsOkIncludeTrue
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterContainerTest.java
CLEANER: de.strullerbaumann.visualee.filter.boundary.TestGroup100Case0.test249
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.boundary.FilterContainerTest.testIsOkIncludeFalse
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterContainerTest.java
ORIGINAL CLEANER SIZE: 16
NEW CLEANER SIZE: 2
ELAPSED TIME: 109.00799989700317
==========================
@@ -113,0 +113,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
